GovUkDesignSystemComponents/SummaryListRowViewModel.cs

```diff
@@ -0,0 +1,117 @@
+﻿using GovUkDesignSystem.GovUkDesignSystemComponents.SubComponents;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GovUkDesignSystem.GovUkDesignSystemComponents
+{
+    public class SummaryListRowViewModel
```

Comment:
Why not add these classes to SummaryListViewModel.cs?

Reply:
Done, thanks